static/js/drafts.js

```diff
@@ -148,6 +148,9 @@ exports.restore_draft = function (draft_id) {
     }
     compose_actions.start(draft_copy.type, draft_copy);
     compose_ui.autosize_textarea();
+
+    compose.clear_preview_area();
+
```

Comment:
I know this sounds really stupid, but what about having this command right below the `clear_compose` command. Like we do all the tear-down first, and then we restore the draft :stuck_out_tongue:

Reply:
@sampritipanda Sure! That makes sense. I made that changed and pushed it.